tools/fuchsia/fuchsia_libs.gni

```diff
@@ -73,62 +73,18 @@ common_libs = [
 vulkan_dist = "$fuchsia_sdk_base/dist"
 
 vulkan_validation_libs = [
```

Comment:
can you add a comment her e indicating that we should not include other validation layer `.so` files for future reference.